inputPage.php

```diff
@@ -0,0 +1,66 @@
+<?php
+require_once('session.php');
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Add a mouse</title>
+    <?php
+    require_once('HTMLhead.php');
+    ?>
+</head>
+
+<body>
+<nav>
+    <?php
+    require_once('HTMLnav.php');
+    ?>
+</nav>
+<main class="inputPageMain">
+    <h2>Add to the Mouse collection!</h2>
+    <div class = "inputPageDiv">
+        <form class="inputPageForm" action="inputPageCode.php" method="post">
+            <label>
+                Mouse name:
+                <input name="mouseName" type="text" placeholder = "Enter Mouse Name Here" maxlength="100" required/>
+            </label>
+            <label>
+                Brand:
+                <input name="mouseBrand" type="text" placeholder = "Enter Mouse Brand Here"maxlength="50" required/>
+            </label>
+            <label>
+                Weight (g):
+                <input name="mouseWeight" type="text" pattern="[0-9]*" placeholder= "Range: 1-1000 (g)" maxlength="10" required/>
+            </label>
+            <label id="wirelessLabel">
+                Is it Wireless?
+                <div class="wirelessDiv">
+                    <input type="radio" name="wirelessInput" value="yes" required>
```

Comment:
make the value 1 and the no value 0, then your php if statement doesnt have to compare to a string